telegram/utils/request.py

```diff
@@ -224,14 +229,38 @@ def _request_wrapper(self, *args, **kwargs):
         else:
             raise NetworkError('{0} ({1})'.format(message, resp.status))
 
-    def get(self, url, timeout=None):
+    def _parse_retries(self, retries):
+        """
+        Builds a :obj:`telegram.vendor.ptb_urllib3.urllib3.util.retry.Retry object.
+        If it's a positive integer, it's used as the total amount of retries, and a new Retry
+        object with a sane default for backoff (0.1s) is created. Else, retrying is disabled.
+
+        Args:
+            retries (:obj:`int`): Retries parameter.
+
+        Returns:
+          A :obj:`telegram.vendor.ptb_urllib3.urllib3.util.retry.Retry` object, or False if
+          retrying is disabled.
+
+        """
+        # TODO: allow configuring the backoff. Ideally we would just let the programmer use the
+        # Retry class for fine-tuning, but that's inside the "private" urllib3 package.
+
+        if isinstance(retries, int) and retries > 0:
+            return Retry(total=retries, backoff_factor=0.1)
+
+        return False
+
+    def get(self, url, timeout=None, retries=None):
         """Request an URL.
 
         Args:
             url (:obj:`str`): The web location we want to retrieve.
             timeout (:obj:`int` | :obj:`float`): If this value is specified, use it as the read
                 timeout from the server (instead of the one specified during creation of the
                 connection pool).
+            retries (:obj:`int`): If this value is specified, use it as the number of retries for
+                the request (instead of the one specified during creation of the connection pool).
```

Comment:
Maybe add `urllib3_req_kwargs` here as well? (here and the other functions below)